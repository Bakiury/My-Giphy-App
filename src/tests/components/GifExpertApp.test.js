import { shallow } from 'enzyme';
import React from 'react'
import { GifExpertApp } from '../../GifExpertApp';
describe('Pruebas en <GifExpertApp />', () => {
    test('Debe Funcionar correctamente', () => {
        const wrapper = shallow(<GifExpertApp />);

        expect(wrapper).toMatchSnapshot();
    });
});