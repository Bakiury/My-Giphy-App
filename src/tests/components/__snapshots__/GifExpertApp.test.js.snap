// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> Debe Funcionar correctamente 1`] = `
<Fragment>
  <h2>
    My Giphy App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <div>
    <GifGrid
      category="Naruto"
      key="Naruto"
    />
  </div>
</Fragment>
`;
